packages/saber/vue-renderer/lib/index.js

```diff
@@ -276,7 +276,9 @@ class VueRenderer {
               .replace(/^\s+/gm, '')
               .replace(/\n+</g, '<')
               .replace('<div id="_saber"></div>', markup)
+            await this.api.hooks.beforeGeneratePage.promise(context)
             await fs.outputFile(route.outputFilePath, html, 'utf8')
+            await this.api.hooks.afterGeneratePage.promise(context)
```

Comment:
let's also pass `{ html, outputFilePath }` to the hook as the second argument.

Reply:
If I am not mistaken I should name it here `afterGeneratePage: new AsyncSeriesHook(['context', 'name'])` what name should I use?

Reply:
maybe `file`

Reply:
LGTM